xml/issue1203.xml

```diff
@@ -134,6 +143,58 @@ and only if <tt>Ostream</tt> is not an lvalue reference type and is derived from
 </ins></p>
 </blockquote>
 </blockquote>
+</blockquote>
+</discussion>
+
+<resolution>
+<p>This resolution is relative to N4750.</p>
+<p>
+Change <sref ref="[istream.rvalue]"/> as follows:
+</p>
+
+<blockquote><pre>
+template &lt;class <del>charT, class traits</del> <ins>Istream</ins>, class T&gt;
+  <del>basic_istream&lt;charT, traits&gt;&amp;</del> <ins>Istream&amp;&amp;</ins>
+  operator&gt;&gt;(<del>basic_istream&lt;charT, traits&gt;</del> <ins>Istream</ins>&amp;&amp; is, T&amp;&amp; x);
+</pre>
+<blockquote>
+<p>
+-1- <i>Effects:</i> Equivalent to:
+</p>
+<pre>
+<tt>is &gt;&gt; std::forward&lt;T&gt;(x)</tt>
+return <ins>std::move(</ins>is<ins>)</ins>;
+</pre>
+<p>-2- <i>Remarks:</i> This function shall not participate in overload
+resolution unless the expression <tt>is &gt;&gt; std::forward&lt;T&gt;(x)</tt> is
+well-formed<ins>, <tt>Istream</tt> is not an lvalue reference type, and
+<tt>Istream</tt> is derived from <tt>ios_base</tt></ins>.</p>
+</blockquote>
+</blockquote>
+
+<p>
+Change <sref ref="[ostream.rvalue]"/>:
+</p>
+
+<blockquote><pre>
+template &lt;class <del>charT, class traits</del> <ins>Ostream</ins>, class T&gt;
+  <del>basic_ostream&lt;charT, traits&gt;&amp;</del> <ins>Ostream&amp;&amp;</ins>
+  operator&lt;&lt;(<del>basic_ostream&lt;charT, traits&gt;</del> <ins>Ostream</ins>&amp;&amp; os, const T&amp; x);
+</pre>
+<blockquote>
+<p>
+-1- <i>Effects:</i> As if by: <tt>os &lt;&lt; x;</tt>
```

Comment:
We use "Equivalent to:" instead of "As if by:" in new wording.

Reply:
The existing wording says "As if by:"

Reply:
This is *old* wording, Casey. (Apparently the green color overrides my brain.)